libphobos/src/std/math.d

```diff
@@ -5377,15 +5330,15 @@ private:
 
                 /* In the FPU control register, rounding mode is in bits 10 and
                 11. In MXCSR it's in bits 13 and 14. */
-                enum ROUNDING_MASK_SSE = ROUNDING_MASK << 3;
-                immutable newRoundingModeSSE = (newState & ROUNDING_MASK) << 3;
+                enum ROUNDING_MASK_SSE = roundingMask << 3;
+                immutable newRoundingModeSSE = (newState & roundingMask) << 3;
                 mxcsr &= ~ROUNDING_MASK_SSE; // delete old rounding mode
                 mxcsr |= newRoundingModeSSE; // write new rounding mode
 
                 /* In the FPU control register, masks are bits 0 through 5.
                 In MXCSR they're 7 through 12. */
-                enum EXCEPTION_MASK_SSE = EXCEPTION_MASK << 7;
-                immutable newExceptionMasks = (newState & EXCEPTION_MASK) << 7;
+                enum EXCEPTION_MASK_SSE = allExceptions << 7;
+                immutable newExceptionMasks = (newState & allExceptions) << 7;
```

Comment:
Also need to update this for the GNU branch.

Reply:
Yes, I just saw the CI failure ;-)